debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/ValueTypeHelper.java

```diff
@@ -217,7 +217,7 @@ public J9ClassPointer findJ9ClassInFlattenedClassCacheWithSigName(J9ClassPointer
 
 		@Override
 		public boolean isRomClassAValueType(J9ROMClassPointer romClass) throws CorruptDataException {
-			return romClass.modifiers().allBitsIn(J9JavaAccessFlags.J9AccValueType);
+			return !romClass.modifiers().allBitsIn(J9JavaAccessFlags.J9AccClassHasIdentity);
```

Comment:
This may lead to misinterpretation of older core files. In dumps from VMs which predate Valhalla support (where the `J9AccClassHasIdentity` constant is not present), `allBitsIn(0)` will answer `false` and the class will be considered a value type which is just wrong.